### DISCUSSION TITLE
What is the difference between scrobble and sync?

### DISCUSSION THREAD

**Discussion by benwaffle** [trakt-tools/universal-trakt-scrobbler#307] [Q&A] [answered]
I'm confused, and I didn't see anything in the README that explains this

**Comment by dochypermoose** [comment, marked as answer]
Scrobble logs the show as you're watching it. Sync logs shows you've previously watched (using your play history).

**Reply by felipecrs** [reply]
I had the same question, would recommend adding this to the README.
